### DISCUSSION TITLE
GoPro Hero11 - 10-bit with 4:3 resolutions

### DISCUSSION THREAD

**Discussion by Fedess1986** [gopro/labs#697] [Feature Requests]
It will be very nice (and doing an internet search it's also very requested) to have 10-bit color into the 4:3 available resolutions. Would it be implemented at least for Gopro Labs?

**Comment by dnewman-gpsw** [comment]
It can't be implemented in Labs. Add Labs typically doesn't add video modes, as there never been a 10-bit 4:3 modes (nothing to copy.)  Each camera has a limited number of video modes, to assist QA and image tuning. 8:7 supplanted all the 4:3 modes for 10-bit.  I do hear this request, and I expect the product team (that makes these decision have heard it too.) However, 8:7 with a higher bitrate, can do everything a 4:3 mode can do, and more.  I expect this is the reason we aren't likely to see 10-bit 4:3 modes.

**Reply by Fedess1986** [reply]
Thank you for the quick reply. I understand that. 
I find (little) benefit when using 2.7k 4:3 at 120fps (limited to 60fps when using 8:7 in Hero11), but I also find useful as it will not push the hardware hard when 2.7k 4:3 at 60fps (plus 10 bit color) could be fine (also avoiding camera overheating and powering off, if a long shot is needed). Probabbly, the overheating thing is out of the scope of Labs.
